test/Table_Tests/src/IO/Excel_Spec.enso

```diff
@@ -1151,6 +1152,18 @@ add_specs suite_builder =
                 r3.at "Value" . at 0 . should_be_a Table
                 r3.at "Value" . at 1 . should_be_a Excel_Workbook
 
+        group_builder.specify "should clear cache on simulated reload" <|
+            ExcelConnectionPool.INSTANCE.simulateReloadTestOnly
+            check_workbook <| xlsx_sheet.read
+            ExcelConnectionPool.INSTANCE.getConnectionRecordCount . should_equal 1
+            check_workbook <| xls_sheet.read
+            ExcelConnectionPool.INSTANCE.getConnectionRecordCount . should_equal 2
```

Comment:
Maybe it would be good to also read `xls_sheet` one more time _before_ the reload and see that it does not increase the counter?

Reply:
Done